.demo/env/charts/cert-manager.yaml

```diff
@@ -1,5 +1,6 @@
 charts:
     cert-manager:
+        _expert: {}
```

Comment:
I am just curios why use private variable naming for the `expert` property?
If that is just to highlight these properties as privately scoped and can be used (or set) only in by certain "actors", then maybe that should be ok but it needs to be documented somewhere.

Reply:
It is a special property that is deemed "unsafe" but exists to have  hard override mechanism for chart values.

Reply:
But we agreed to just call it `_values` @j-zimnowoda

Reply:
I amend it first `_values`. Then I realise that it is not meaningful - it is too general. Everything is values.
We always talk about expert mode the why not calling it like that?

Reply:
_rawValues it is

Reply:
call things by their name...the fact that in "expert" mode we use raw values is not conflicting, it is more precise. The values themselves are not "expert" in any way

Reply:
`_rawValues` best choice  👍

Reply:
done

Reply:
> I am just curios why use private variable naming for the `expert` property?
> If that is just to highlight these properties as privately scoped and can be used (or set) only in by certain "actors", then maybe that should be ok but it needs to be documented somewhere.

We provide some sane values, but with this property users have much more freedom for setting chart values.